packages/manager/modules/pci/src/projects/project/users/download-openrc/download-openrc.controller.js

```diff
@@ -27,12 +27,13 @@ export default class PciUsersDownloadOpenRcController {
       label: this.CucRegionService.getTranslatedMicroRegion(region),
     }));
     this.region = first(this.regions);
+    this.hasGlobalRegions = this.PciProjectsProjectUsersService.checkGlobalRegion(this.regions);
   }
 
   downloadOpenRc() {
     this.isLoading = true;
     return this.PciProjectsProjectUsersService
-      .downloadOpenRc(this.projectId, this.user, this.region.id, this.useVersion3 ? 3 : 2)
+      .downloadOpenRc(this.projectId, this.user, this.region.id, this.hasGlobalRegions ? 3 : 2)
```

Comment:
Maybe the `this.hasGlobalRegions ? 3 : 2` could be turned to a `getVersion` function ? At first look it's not easy to understand what those number correspond to